### DISCUSSION TITLE
Error Running a Local Android Build

### DISCUSSION THREAD

**Discussion by NinaSpAce** [bluesky-social/atproto#3239] [Dev Debugging]
Building the local Android environment for the first time using "yarn android." But this error keeps appearing.

Error: C:\Users\ninam\source\repos\social-app\android\gradlew.bat app:assembleDebug -x lint -x test --configure-on-demand --build-cache -PreactNativeDevServerPort=8081 -PreactNativeArchitectures=x86_64,arm64-v8a exited with non-zero code: 1

Not sure whats happening, keep seeing log messages about Gradle or Kotlin not supporting the versions I have. 

![image](https://github.com/user-attachments/assets/d0e3dc56-4c77-41be-a56c-da1409bda852)

Thanks for reading and responding!

**Comment by bnewbold** [comment]
Please open as an issue over on the `social-app` repo.
